### ISSUE TITLE
Fatal Error When Adding a Place

### ISSUE THREAD

**Issue by miqrogroove** [collectiveaccess/providence#1349]
Rather than add a Place from the New menu, I was trying to use the Lists editor to add to the place_hierarchies list.  This didn't go well.

```
[23-Jun-2022 11:27:06 America/New_York] PHP Fatal error:  Uncaught Error: Class 'ca_places' not found in app/models/ca_list_items.php:624
Stack trace:
#0 app/lib/BundlableLabelableBaseModelWithAttributes.php(3662): ca_list_items->insert()
#1 app/lib/BaseEditorController.php(302): BundlableLabelableBaseModelWithAttributes->saveBundlesForScreen()
#2 app/lib/Controller/RequestDispatcher.php(273): BaseEditorController->Save()
#3 app/lib/Controller/AppController.php(113): RequestDispatcher->dispatch()
#4 index.php(120): AppController->dispatch()
#5 {main}
  thrown in app/models/ca_list_items.php on line 624
```

**Comment by collectiveaccess** [comment]
A class wasn't preloaded. I've added code to load it in this context. It's in master (aka 1.7.16) now, along with some other fixes. Can you try it out?

**Comment by miqrogroove** [comment]
https://github.com/collectiveaccess/providence/commit/4ae20e09fe72c9dc29b8bbac7560fa0452e7716e

This works.

**Comment by miqrogroove** [comment]
Fixed by 4ae20e0
